src/ui/components/Rating/index.js

```diff
@@ -60,7 +60,7 @@ export class RatingBase extends React.Component {
         return i18n.sprintf(i18n.gettext('Rated %(rating)s out of 5.'),
           { rating: i18n.formatNumber(parseFloat(rating).toFixed(1)) });
       }
-      return i18n.gettext('This add-on has not been rated yet.');
+      return i18n.gettext('There are no ratings yet.');
```

Comment:
Can you remove the period please? See: https://design.firefox.com/photon/copy/punctuation.html

If you could do that for the other texts in the `renderTitle()` method, that would be ❤️